gateway-ha/src/main/java/com/lyft/data/gateway/ha/router/PrestoResourceManager.java

```diff
@@ -0,0 +1,82 @@
+package com.lyft.data.gateway.ha.router;
+
+import java.util.List;
+import lombok.Data;
+import lombok.ToString;
+
+public interface PrestoResourceManager {
```

Comment:
this should be `ResourceGroupsManager` as it defines what operations are allowed for ResourceGroups management.